.github/workflows/release.yaml

```
@@ -149,4 +149,33 @@ jobs:
           name: v${{ env.VERSION }}
           draft: true
           prerelease: ${{ startsWith(env.VERSION, '0') }}
-          files: artifacts/*
\ No newline at end of file
+          files: artifacts/*
+
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    needs: [ prepare ]
+    env:
+      VERSION: ${{ needs.prepare.outputs.version }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: https://ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
```

```suggestion
          platforms:
            - linux/amd64
            - linux/arm64
```

Let's use YAML's list notation here instead.

---

I committed this but realised this isn't possible. `platforms` expects a single string, so I'll be reverting this. [Specification here](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepswith), sorry about that.